policyengine_us/variables/household/demographic/person/public_support_received.py

```diff
@@ -0,0 +1,8 @@
+from policyengine_us.model_api import *
+
+
+class public_support_received(Variable):
+    value_type = bool
+    entity = Person
+    label = "Child receiving public support"
```

Comment:
what does this mean?

Reply:
I mean how do they define "public support"?

The legal code says:
>For purposes of claiming this credit only, a minor child receiving support from the department of human services of the State, social security survivor’s benefits, and the like, may be considered a dependent and a qualified exemption of the parent or guardian.

The tax form says:
>On line 3, list your minor children who are also qualified exemptions, and who receive more than half of their support from the Department of Human Services, Social Security benefits, and other government payments

We can check if they receive `social_security_survivor_benefits` or `tanf`. I don't know how we would define the denominator though; I think they might be referring to `child_support_received` as the other form of support. So I'd make a new bool variable for `hi_food_excise_credit_child_receiving_public_support` with that formula and a comment indicating our interpretation of the law. Also need a parameter for `0.5`.